src/main/java/com/icthh/xm/ms/entity/lep/XmEntityMsLepProcessingApplicationListener.java

```diff
@@ -127,6 +130,7 @@ protected void bindExecutionContext(ScopedContext executionContext) {
 
         executionContext.setValue(BINDING_METRICS_CONTEXT, customMetricsContext);
 
+        executionContext.setValue(BINDING_KEY_METER_REGISTRY, meterRegistry);
```

Comment:
Pls, do not put third party class to lep context. Wrap meterRegistry to custom service.
Why don't use customMetricsContext and custom metrix feature?

Reply:
why you don't use https://github.com/xm-online/xm-online/wiki/How-to-add-custom-tenant-specific-metrics ?